python/ray/rllib/models/catalog.py

```diff
@@ -54,6 +54,21 @@ def get_action_dist(action_space, dist_type=None):
                 return Deterministic, action_space.shape[0]
         elif isinstance(action_space, gym.spaces.Discrete):
             return Categorical, action_space.n
+        elif isinstance(action_space, list):
```

Comment:
Does it make sense to push this down to the Catalog level? I think it would be reasonable if this were done as a helper outside of this function.

Reply:
Could you explain what you mean by Catalog level?

Reply:
I'm not sure what Catalog level means, but it seems like the "distribution list" here can be kept inside an ActionDistribution class (e.g. if you call sample() internally it would sample from N distributions and concat them). That way, you wouldn't need to special case handling of this distribution since it would behave like a normal action distribution. Let me know if that makes sense.

Reply:
Oh I meant "Catalog level' as in modifying the Catalog code; alternative, you could have the Evaluator take care of this by something along the lines of `list_of_dists =
 [ModelCatalog.get_action_dist(space) for space in vectorized_action_space`

Reply:
Yeah the issue there is that now you have to special case the action distribution being a list everywhere.

Reply:
Makes sense, I'll do it this way.